.github/actions/test-deploy-lambda-function/action.yml

```
@@ -0,0 +1,54 @@
+---
+
+name: 'Test deploy-to-lambda-function action'
+description: 'Runs validation against the deploy-to-lambda-function action'
+
+inputs:
+  aws-account-id:
+    description: 'The AWS Account id'
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    - name: 'Setup brew'
+      uses: Homebrew/actions/setup-homebrew@master
+
+    - name: 'Install BATS'
+      shell: bash
+      run: brew install bats-core
+
+    - name: 'Checkout'
+      uses: actions/checkout@v3
+
+    - name: 'Use local actions'
+      uses: ./.github/actions/use-local-actions
+
+    - name: 'Create archive file'
+      shell: bash
+      working-directory: ${{ github.action_path }}
+      run: |
+        sed -i "s/const VERSION = 'dev'/const VERSION = '${{ github.sha }}'/" index.js
+        zip archive.zip ./index.js
+
+    - name: 'Run deploy-lambda-function action'
+      id: unpack
+      uses: ./actions/deploy-lambda-function
+      with:
+        pattern: ${{ github.action_path }}/archive.zip
+        function-name: shopsmart-github-actions-tests
+        function-tags: version=${{ github.sha }}
+        role-to-assume: arn:aws:iam::${{ inputs.aws-account-id }}:role/github-actions-tests
+
+    - name: 'Configure AWS Credentials'
+      uses: aws-actions/configure-aws-credentials@v1
```

```suggestion
      uses: aws-actions/configure-aws-credentials@v1-node16
```

Removes deprecation warning